test/modules/image-sequencer.js

```diff
@@ -184,7 +184,7 @@ test('getStep(offset) returns the step at offset distance relative to current st
 });
 
 test('toCliString() returns the CLI command for the sequence', function(t) {
-  t.deepEqual(sequencer.toCliString(), `sequencer -i [PATH] -s "channel channel channel channel channel invert blend" -d '{"channel":"green"}'`, "works correctly");
+  t.deepEqual(sequencer.toCliString(), `sequencer -i [PATH] -s "channel channel channel channel channel invert blend" -d '{"channel":"green","offset":-2}'`, "works correctly");
```

Comment:
Here, perhaps we should introduce a new test for offsets, and we could do a test demonstrating that the default offset and `-3` don't return the same result. You can run a sequence that inverts repeatedly a few times, then does blend, and this kind of test should work. You could say `-3` isn't the same as `-2`, but `-4` is the same, you know? And create a new test specifically for this functionality.

Reply:
I'll think about it and try to add new test soon.

Reply:
Is there some kind of documentation on how to do tests? It's time consuming by doing a lot of trials, espiecially when I can't run it locally, and each time I have to run in on Travis Ci, becouse there are some errors on the windows.

Reply:
ah, have you had trouble running tests locally? I think you should be able to with `npm test` -- but if you did something like:

```js
t.notDeepEqual(sequencer.toCliString(), `sequencer -i [PATH] -s "invert invert blend" -d '{"channel":"green","offset":-2}'`, "works correctly");
```

Docs are here: https://github.com/substack/tape

Tutorial: https://ponyfoo.com/articles/testing-javascript-modules-with-tape

But you're right, we should add these to a `Testing` section of the README -- would you be interested in adding a section like that? Otherwise I can open an issue for someone new to try it out. 

Thanks @KusioDev !! This is going to be awesome.

Reply:
I wanted to write test that invert twice first, and then blend. 
It saves the first output to variable, change the offset to -3, and re-run drawing the blend. 
However, when I run `sequencer.run()` after changing the offset, the outuput is set to undefined, and so the test is not correct, becouse I can't read property of `.output.src` if output is not set.